RNSensitiveInfo.js

```diff
@@ -1,3 +1,21 @@
-import { NativeModules } from 'react-native';
+import { NativeModules } from "react-native";
 
-module.exports = NativeModules.RNSensitiveInfo;
+const RNSensitiveInfo = NativeModules.RNSensitiveInfo;
+
+module.exports = {
+  ...RNSensitiveInfo,
+  setInvalidatedByBiometricEnrollment() {
```

Comment:
Since that method is not available on iOS it's needed to add a guard in user code, but I think it's better to handle that in the library. (At least we have them on TS typings, and can separate that for different OS).

Btw, there are isHardwareDetected() and hasEnrolledFingerprints() methods, that needed to be guarded in a same way, but it will be unfair to do nothing, and it will be hard to debug, why nothing happened.